tests/cli_test.py

```diff
@@ -360,33 +344,87 @@ def cleos_abi_file_test():
+        node.verifyAlive() # Setting node state to not alive
+        node.relaunch(newChain=True, cachePopen=True)
+        accountNames = ["eosio", "eosio.token", "alice", "bob"]
+        accounts = []
+        for name in accountNames:
+            account = Account(name)
+            account.ownerPrivateKey = account.activePrivateKey = "5KQwrPbwdL6PhXujxW37FSSQZ1JiwsST4cqQzDeyXtP79zkvFD3"
+            account.ownerPublicKey = account.activePublicKey = "EOS6MRyAjQq8ud7hVNYcfnVPJqcVpscN5So8BhtHuGYqET5GDW5CV"
+            accounts.append(account)
+        walletMgr.create('eosio', [accounts[0]])
+        node.createAccount(accounts[1], accounts[0], stakedDeposit=0)
+        node.publishContract(accounts[1], contractDir, 'eosio.token.wasm', 'eosio.token.abi')
+        account = 'eosio.token'
+        action = 'create'
+        data = '{"issuer":"eosio.token","maximum_supply":"100000.0000 SYS","can_freeze":"0","can_recall":"0","can_whitelist":"0"}'
+        permission = '--permission eosio.token@active'
+        node.pushMessage(account, action, data, permission)
+        action = 'issue'
+        data = '{"from":"eosio.token","to":"eosio.token","quantity":"100000.0000 SYS","memo":"issue"}'
+        node.pushMessage(account, action, data, permission)
+        node.createAccount(accounts[2], accounts[0], stakedDeposit=0)
+        node.createAccount(accounts[3], accounts[0], stakedDeposit=0)
+
+        node.transferFunds(accounts[1], accounts[2], '100.0000 SYS')
+
+        node.processCleosCmd('set abi eosio.token ' + malicious_token_abi_path, 'set malicious eosio.token abi', returnType=ReturnType.raw)
+
+        cmdArr = node._Node__transferFundsCmdArr(accounts[2], accounts[3], '25.0000 SYS', 'm', False, None, False, False, 90, False)
+        cmdArr.insert(6, '--print-request')
+        cmdArr.insert(7, '--abi-file')
+        cmdArr.insert(8, token_abi_file_arg)
+        Utils.runCmdArrReturnStr(cmdArr)
+        balance = node.getCurrencyBalance('eosio.token', 'alice')
+        assert balance == '75.0000 SYS\n'
+        testSuccessful=True
+    except Exception as e:
+        testSuccessful=False
+        Utils.Print(e.args)
     finally:
-
-        if pNodeos.pid:
-            os.kill(pNodeos.pid, signal.SIGKILL)
-        shutil.rmtree(data_dir)
-
+        if testSuccessful:
+            Utils.Print("Test succeeded.")
+        else:
+            Utils.Print("Test failed.")
+        if node:
+            if not node.killed:
+                if node.pid:
+                    os.kill(node.pid, signal.SIGKILL)
+        if testSuccessful:
+            Utils.Print("Cleanup nodeos data.")
+            shutil.rmtree(data_dir)
+
+        '''
```

Comment:
any reason to comment this out instead of removing it?

Reply:
Forgot this was here after removing other commented code.  Removed.